testcases/misc_tests/cca_export_import_test.c

```diff
@@ -1259,6 +1259,11 @@ static CK_RV cca_rsa_export_import_tests(void)
         priv_opaquekey = NULL;
         free(publ_opaquekey);
         publ_opaquekey = NULL;
+
+        if (publ_key != CK_INVALID_HANDLE)
+            funcs->C_DestroyObject(session, publ_key);
+        if (priv_key != CK_INVALID_HANDLE)
+            funcs->C_DestroyObject(session, priv_key);
```

Comment:
Shouldn't also the `imp_priv_key` and `imp_publ_key` keys that are created in line 1155 and 1177 be destroyed here? Those are session objects, so you won't see them in the repository afterwards, but when we are at it, lets destroy them, too.

Reply:
Yes, ok, will do ...

Reply:
You still miss to destroy imp_priv_key and imp_publ_key in cca_rsa_export_import_tests(). You are destroying it in cca_ecc_export_import_tests(), but still not in cca_rsa_export_import_tests().

Reply:
Yep, corrected.